app/assets/javascripts/backbone/views/grumblesListView.js

```diff
@@ -0,0 +1,18 @@
+var GrumblesListView = Backbone.View.extend({
+  el: "#grumble-list",
+
+  initialize: function(){
+    console.log( "GrumblesListView initialized!" );
+    this.listenTo( this.collection, "add", this.renderAll );
+    this.listenTo( this.collection, "reset", this.renderAll );
+  },
+  renderOne: function( grumble ){
+    var currentView = new GrumbleView( { model: grumble });
+    this.$el.append( currentView.$el );
+  },
+  renderAll: function(){
+    this.$el.empty();
+    var theView = this;
+    this.collection.each( this.renderOne.bind( theView ) );
+  }
```

Comment:
im curious why you decided to declare a new variable on line 15 instead of just passing in `this` to the bind.